app/access/api.py

```diff
@@ -42,3 +45,22 @@ def users(request: HttpRequest) -> dict[str, list[UserSchema]]:
     models = User.objects.all()
     responses = [user_to_response(model) for model in models]
     return {"items": responses}
+
+
+@router.delete("users/{username}")
+def delete(request: HttpRequest, username: str) -> HttpResponse:
+    """
+    Delete the user with the given username.
+
+    Return HTTP status code
+    - 204 (No Content) if the user has been deleted
+    - 404 (Not Found) if there is no user with the given username
+    - 500 (Internal Server Error) if there is inconsistency with cognito
+    - 503 (Service Unavailable) if cognito cannot be reached
+    """
+    model = get_object_or_404(User, username=username)
```

Comment:
I'd name this `user` or `object` since we don't delete the model itself, rather an object or instance of a model

Reply:
It is named `model` for consistency with the other API functions in this file. I don't mind renaming this, but then we also need to:
- rename `model`-variables in the other functions for consistency (probably best to do this in the other APIs as well)
- rename the API endpoint functions (since the get endpoint is also called `user`; probably best to do this in the other APIs as well)

Reply:
Renamed